guacamole/src/main/webapp/app/login/directives/login.js

```diff
@@ -102,6 +102,13 @@ angular.module('login').directive('guacLogin', [function guacLogin() {
          */
         $scope.submitted = false;
 
+        /**
+         * The field that needs to be focused
+         *
+         * @type Field
+         */
+        $scope.relevantField = {};
```

Comment:
`{}` is not a `Field` and will also fail simple tests for whether the value has been set (`{}` is truthy). If you're looking for a default value prior to actual assignment later, `null` would be a better choice.